include/rocksdb/options.h

```diff
@@ -683,6 +683,12 @@ struct DBOptions {
   // Default: 600
   unsigned int stats_persist_period_sec = 600;
 
+  // If true, automatically persist stats to a hidden column family every
```

Comment:
Need to mention the name of the hidden column family name here. Also need to mention in case a user provide a name of the name, what the behavior they are expected to see.

Reply:
In `DBImpl::Recover` where we recover the manifest, how do we tell if the hidden column family was explicitly created by user or implicitly created by persistent stats feature? 
I imagine if we want to disable persistent stats when user provided column family name conflicts with hidden column family name, we need to be able to tell the difference?